src/univers/version_range.py

```diff
@@ -260,6 +261,37 @@ def from_native(cls, string):
 
         return cls(constraints=anyof_constraints)
 
+    @classmethod
+    def from_gitlab_native(cls, string):
```

Comment:
A lot of the code seems to be duplicated in the other "from_gitlab_native" method below?
Could there be some shared function to avoid some duplication and still keep things readable?